_commands/basics/id.md

```diff
@@ -16,9 +17,33 @@ ToDo: Needs completion
 
 #### Example command
 
-##### Break it down
+to find a specific user's UID, add -u {username}. Moreover, id -G USERNAME will find all the groups a user belongs to. Force id to display the name of the UID or GID instead of ugly numbers by passing the -n options
```

Comment:
To match the style of others, can you add backticks around all of the command line stuff in in-line text, i.e. `-u {username}` and `id -G USERNAME`